backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -220,7 +220,7 @@ private ApiUser getApiUser(UUID id) {
   }
 
   public boolean isAdmin(ApiUser user) {
```

Comment:
We should seek to get rid of this method rather than re-implement it. 🙂

Reply:
Would it make sense to have a `SITE_ADMIN` user role (and maybe an associated permission) instead?

Reply:
Probably yes on the roles and no on the permissions? I started to think about this a while back and my head started to hurt a little so I stopped.

The issue is that the main place where we parse roles, we are analyzing them for organization membership, and grouping the claims by what organization they relate to, then making sure there's only one active organization involved. The global admin role doesn't really fit in with those. I was expecting us to eventually have at least one other global role that we might grant, which would give some insight into how to generalize the problem, but that hasn't happened, so I'm still kind of 🤷  about how it should all fit together.